web-editor/src/screen/Home.jsx

```diff
@@ -75,62 +84,144 @@ const pages = [
 ];
 
 export const Home = (props) => {
+  const [option, setOption] = useState("Design your own");
+  const copyright = new Date().getFullYear() + " Muxable, LLC";
```

Comment:
```suggestion
  const copyright = new Date().getFullYear() + " Muxable, Inc";
```

I know it says LLC on the mocks but we're a C-Corp.